manifests/images/mfsbsd.pp

```diff
@@ -51,11 +51,13 @@
   }
 
   $tftp_root = $::pxe::tftp_root
+  $imgdir    = "${tftp_root}/images/${os}/${ver}/${arch}"
+  $imgfile   = "${os}-${ver}-${arch}.img"
 
   exec { "wget ${os} live image ${arch} ${ver}":
     path    => ['/usr/bin', '/usr/local/bin'],
-    cwd     => "${tftp_root}/images/${os}/${ver}/${arch}",
+    cwd     => $imgdir,
     command => "wget ${srclocation}/${path}",
-    creates => "${tftp_root}/images/${os}/${path}";
+    creates => "${imgdir}/${imgfile}",
```

Comment:
This value doesn't look correct to me.  When `wget` is called, the created file will likely match the source file name, so the creates here won't match `$imagefile` unless we specify `$imagefile` as the target filename in the call to `wget`.